src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -17,14 +20,19 @@
 public class LineService {
 
     private final LineRepository lineRepository;
+    private final SectionService sectionService;
 
-    public LineService(LineRepository lineRepository) {
+    public LineService(LineRepository lineRepository, SectionService sectionService) {
         this.lineRepository = lineRepository;
+        this.sectionService = sectionService;
     }
 
     public LineResponse saveLine(LineRequest request) {
-        Line persistLine = lineRepository.save(request.toLine());
-        return LineResponse.of(persistLine);
+        Line line = lineRepository.save(request.toLine());
+        Section section = sectionService.saveSectionWith(line, request);
+        line.add(section);
```

Comment:
```suggestion
        line.addSection(section);
```
이렇게 표현할 수 있도록 작업해보면 어떨까요?